### PR TITLE
Rewrite ssh user to ubuntu in bastions docs

### PR DESCRIPTION
**Pull request by h3poteto**
Refs: #9751 
I only update a doc `kops-tests-private-net-bastion-host`.

### CODE REVIEW

**Comment by hakman** [comment]
@h3poteto Hello again. In this case, I think we may want to update all docs from `admin@` to `ubuntu@`. There maybe be some other references to Debian in those places that should be updated also.
Would this be ok for you?

**Comment by h3poteto** [comment]
Hi, OK, I see.
I will update all user from `admin@` to `ubuntu@`.

**Comment by hakman** [comment]
Thanks @h3poteto

**Comment by h3poteto** [comment]
@hakman I updated `kops-tests-private-net-bastion-host`.
Should I rewrite ssh user in other documents as well?
I confirm that `admin` user is used for ssh in some documents, but I haven't confirmed if they are the correct information yet.

File: docs/examples/kops-tests-private-net-bastion-host.md [hunk 1]
```diff
@@ -148,8 +148,8 @@ kind: InstanceGroup
 metadata:
   name: bastions
 spec:
-  image: kope.io/k8s-1.7-debian-jessie-amd64-hvm-ebs-2017-07-28
-  machineType: t2.micro
+  image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20200907
```
Comment (hakman) [inline docs/examples/kops-tests-private-net-bastion-host.md:151]: There is another reference to a `kope.io` image in the file. Please update that one too.
Also, please change `t2.micro` to `t3.micro` everywhere.

**Review by hakman — CHANGES_REQUESTED** [review]
Let's keep this commit focused on this file.The rest of the `admin@` can be updated in a followup PR.

File: docs/examples/kops-tests-private-net-bastion-host.md (hunk 1 above)
Comment (h3poteto) [reply docs/examples/kops-tests-private-net-bastion-host.md:151]: I changed it in https://github.com/kubernetes/kops/pull/10044/commits/aff8c0af0090bb8e61c9c11a55c418547b95082b

**Comment by hakman** [comment]
Great, thanks @h3poteto !
/lgtm
/approve

**Pull request merged**